textext.py

```diff
@@ -279,7 +290,7 @@ def preview_convert(text, preamble_file, converter_class, image_setter):
             os.chdir(cwd)
             converter.finish()
 
-    def do_convert(self, text, preamble_file, scale_factor, converter_class, old_node):
+    def do_convert(self, text, preamble_file, scale_factor_from_user, converter_class, old_node):
```

Comment:
I was thinking about maybe renaming this parameter to user_scale_factor instead?

Reply:
Good idea, will change it.